docker-compose.yml

```diff
@@ -8,3 +7,15 @@ services:
       SA_PASSWORD: P@ssword1
     ports:
       - "1455:1433"
+
+  minio:
```

Comment:
Did you consider using moto for s3 tests? It's easy to use and doesn't require running a container in the background (see http://docs.getmoto.org/en/latest/docs/server_mode.html#start-within-python).

Reply:
cf. https://github.com/pola-rs/polars/pull/24443